simplivity/resources/omnistack_clusters.py

```diff
@@ -116,3 +116,19 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+
+    def get_connected_clusters(self):
+        """Retrieves directly connected omnistack_clusters.
+
+        Returns:
+            list: List of omnistack_clusters objects.
+        """
+        method_url = "{}/{}/connected_clusters".format(URL, self.data["id"])
+        connected_clusters = self._client.do_get(method_url).get("omnistack_clusters", [])
+
+        cluster_objs = []
```

Comment:
The pythonic way to write this would be this.

```clusters = [ self._clusters.get_by_id(cluster["id"]) for cluster in connected_clusters]```

Reply:
Updated as per suggestion.